Sming/SmingCore/Timer.h

```diff
@@ -154,28 +164,28 @@ class Timer
 	void IRAM_ATTR setCallback(const TimerDelegateStdFunction& delegateFunction);
 
 protected:
-	static void IRAM_ATTR processing(void* arg);
+	void IRAM_ATTR processing();
 
 	/** @brief  virtual timer loop() method
      *  @note   Can be override in class derivations. If overwriten,
      *          no classic other callbacks are working.
      */
-	virtual void tick();
+	void tick();
 	/** @} */
 
 private:
-	os_timer_t timer;
-	uint64_t interval = 0;
+	OSTimer osTimer; ///< We use the OSTimer class to access the hardware
+	uint32_t interval = 0;
```

Comment:
Is decreasing the timer precision from uint64_t to uint32_t correct?

Reply:
Yes, it only stores small (32-bit) counts, the longIntervalXXX variables deal with the rest. See `setIntervalUs()`.